layer/profiles.cpp

```diff
@@ -7034,7 +7034,7 @@ VKAPI_ATTR VkResult VKAPI_CALL CreateInstance(const VkInstanceCreateInfo *pCreat
     for (uint32_t i = 0; i < pCreateInfo->enabledExtensionCount; ++i) {
         strcpy(extension_names[i], pCreateInfo->ppEnabledExtensionNames[i]);
     }
-    extension_names[pCreateInfo->enabledExtensionCount] = VK_KHR_GET_PHYSICAL_DEVICE_PROPERTIES_2_EXTENSION_NAME;
+    strcpy(extension_names[pCreateInfo->enabledExtensionCount], VK_KHR_GET_PHYSICAL_DEVICE_PROPERTIES_2_EXTENSION_NAME);
```

Comment:
This is really broken because the typo of `std::vector<char *> extension_names` is misleading and wrong.

It should be `std::vector<const char *> extension_names`. it's an array of strings, not an array of characters